### PR TITLE
ENG-16589 double free

### PR DESCRIPTION
**Pull request by luyangco**

### CODE REVIEW

File: src/frontend/org/voltdb/export/ExportDataSource.java [hunk 1]
```diff
@@ -1403,6 +1405,13 @@ public void remoteAck(final long seq) {
             @Override
             public void run() {
                 try {
+                    if (!isMaster()) {
```
Comment (wweiss-voltdb) [inline src/frontend/org/voltdb/export/ExportDataSource.java]: This does not match the debug log below. We should ignore a remote Ack if we are the master.

Comment (luyangco) [reply src/frontend/org/voltdb/export/ExportDataSource.java]: Fixed.

**Comment by luyangco** [comment]
Ok, the potential race condition on pending buffer causes some "double free" complaints,  I reproduced the this symptom twice. But I guess there would also be other race conditions I've not found. I'll create a new ticket and send pull request under new ENG number.

**Comment by luyangco** [comment]
Ticket ENG-16908 is created to track the issue. Close this pull request.

**Pull request closed without merging**